packages/react/src/chakra-provider.tsx

```diff
@@ -72,8 +73,14 @@ export const ChakraProvider = (props: ChakraProviderProps) => {
     cssVarsRoot,
   } = props
 
-  return (
+  const _children = (
```

Comment:
If you want to conditionally wrap I can highly recommend sth like

```
const ConditionalWrapper = ({ condition, wrapper, children }) => 
  condition ? wrapper(children) : children;
  ```
  
  
  ```tsx
<ConditionalWrapper
    condition={portalZIndex}
    wrapper={(children) => <PortalManager zIndex={portalZIndex}>{children}</PortalManager>}
>
    <EnvironmentProvider environment={environment}>{children}</EnvironmentProvider>
</ConditionalWrapper>
 ```

Reply:
That looks interesting. Not a fan of those wrappers but I'll consider it.

Thanks for sharing 🙏